syntax = "proto3";
option go_package = ".;icon";

enum IconID {
    CHECK = 0;
    BOX = 1;
    CROSS = 2;
}

message IconArg {
    IconID icon_id = 2;
}
